packages/web/docs/pages/self-hosting/get-started.mdx

```diff
@@ -0,0 +1,122 @@
+import Callout from 'nextra-theme-docs/callout'
+
+# Self-Hosting GraphQL Hive
+
+If you can not use the GraphQL Hive SaaS service, you can self-host it.
+The self-hosted version is free and open-source.
+You can find the full source code on GitHub.
+
+<Callout>
+  **Self-hosting GraphQL Hive** does currently not support all of the features as the SaaS version.
+
+The following features are currently unavailable when self-hosting:
+
+- CDN (high availability integration)
+- Billing (payment integration)
+- Usage rate-limiting
+
+</Callout>
+
+## Pre-Requisites
+
+The easiest way of running GraphQL Hive is using [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/).
+
+All the services required for running GraphQL Hive are published to the GitHub container registry ([GraphQL Hive Docker Images](https://github.com/kamilkisiela?tab=packages&repo_name=graphql-hive)).
+
+Please make sure to install the Docker daemon as well as `docker-compose` on your machine before proceeding.
+
+## Running GraphQL Hive
+
+First download the `docker-compose.community.yml` file from the [GitHub repository](https://github.com/kamilkisiela/graphql-hive/blob/main/docker-compose.community.yml) using `wget`.
+
+```bash
+wget https://github.com/kamilkisiela/graphql-hive/blob/main/docker-compose.community.yml
+```
+
+After downloading the file, you need to set some environment variables.
+
+<Callout>
+  **GitHub.** For receiving the `GITHUB_APP_ID` and `GITHUB_APP_PRIVATE_KEY` values follow the [Creating a GitHub
+  App](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app) documentation.
+</Callout>
+
+<Callout>
```

Comment:
Are you sure slack is required? I think if a user does not click "Connect Slack" in org settings then it's fine and everything works.

Reply:
we can make it optional afterwards - though it shouldnt block this PR from advancing?